src/core/src/core_logic/PatchInstaller.py

```diff
@@ -70,6 +71,7 @@ def start_installation(self, simulate=False):
                 self.composite_logger.log_debug("Attempting to reboot the machine prior to patch installation as there is a reboot pending...")
                 reboot_manager.start_reboot_if_required_and_time_available(maintenance_window.get_remaining_time_in_minutes(None, False))
 
+        startTime = self.env_layer.datetime.datetime_utcnow()
         # Install Updates
         installed_update_count, update_run_successful, maintenance_window_exceeded = self.install_updates(maintenance_window, package_manager, simulate)
```

Comment:
nit note: this is a remanent of v1 -- this variable should have ideally been patch_installation_operation_successful

The nomenclature changed from "update" to "patch" from V1 to V2. "Run" became "operation". We aim to have consistency from the top of the stack to the bottom in naming for code and service quality.

You don't have to address this, but you can if you want to.